packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/create/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "octavia_load_balancer_health_monitor_title": "Ajouter un health monitor",
+  "octavia_load_balancer_health_monitor_description_1": "Le moniteur d’intégrité est utilisé pour déterminer l’intégrité des membres de votre pool. Des contrôles d’intégrité sont exécutés régulièrement sur chaque membre du pool et le résultat de ces contrôles est utilisé pour déterminer si le membre reçoit de nouvelles connexions. Un seul moniteur d’intégrité peut être défini pour chaque pool.",
```

Comment:
I guess this translation comes from the ticket, but I think "Health Monitor" shouldn't be translated into "moniteur d'intégrité"